service/reportsStorage/storage.go

```diff
@@ -0,0 +1,361 @@
+package reportsStorage
+
+import (
+	"bytes"
+	"context"
+	"crypto/sha256"
+	"encoding/base64"
+	"fmt"
+	"io"
+	"strings"
+	"time"
+
+	"github.com/agext/uuid"
+	"github.com/go-openapi/strfmt"
+	"github.com/rs/zerolog"
+
+	"github.com/iryonetwork/wwm/gen/reports/models"
+	storageModels "github.com/iryonetwork/wwm/gen/storage/models"
```

Comment:
could not import github.com/iryonetwork/wwm/gen/storage/models (cannot find package "github.com/iryonetwork/wwm/gen/storage/models" in any of:
	/tmp/tmp.qkYKz9nbyP/src/github.com/iryonetwork/wwm/vendor/github.com/iryonetwork/wwm/gen/storage/models (vendor tree)
	/usr/local/go/src/github.com/iryonetwork/wwm/gen/storage/models (from $GOROOT)
	/tmp/tmp.qkYKz9nbyP/src/github.com/iryonetwork/wwm/gen/storage/models (from $GOPATH))